src/desktop/apps/sitemaps/test/routes.coffee

```diff
@@ -68,7 +68,7 @@ Sitemap: https://www.artsy.net/sitemap-videos.xml
         @res.send.args[0][0].slice(-1).should.eql('\n')
 
   describe '#news', ->
-    it 'displays the sitemap for news that are articles < 2 days old', ->
+    it 'displays the sitemap for articles', ->
```

Comment:
I think it would be worthwhile adding tests to confirm that both `/news` and `/article` paths are displayed correctly.

Reply:
I've moved the expectations to [src/desktop/apps/sitemaps/test/templates.coffee](https://github.com/artsy/force/blob/41ad825164c1cb2e895b4fa7f090975f48d40cea/src/desktop/apps/sitemaps/test/templates.coffee#L50-L51) which I think is a more appropriate place.